python/Ganga/GPIDev/Base/Objects.py

```diff
@@ -41,7 +42,7 @@ def _getName(obj):
 
 _imported_GangaList = None
 
-do_not_copy = ['_index_cache', '_parent', '_registry', '_data', '_read_lock', '_write_lock', '_proxyObject']
+do_not_copy = ['_index_cache', '_index_cache_dict', '_parent', '_registry', '_data', '_data_dict', '_read_lock', '_write_lock', '_proxyObject']
```

Comment:
Ok, I will bear that in mind, thanks.

The `memo` dict is used during the `deepcopy` operation to cache any objects which have already been deep copied. It ends up being a mapping of object id (in the Python sense) to the deep-copy of the object. The idea is that if `deepcopy` is called on an object and that object is already in the `memo` dict then it does not recurse and instead just returns the object, thereby preventing infinite loops.